src/My.Extensions.Localization.Json/JsonStringLocalizer.cs

```diff
@@ -26,7 +26,7 @@ public class JsonStringLocalizer : IStringLocalizer
             ILogger logger)
         {
             _resourcesPath = resourcesPath ?? throw new ArgumentNullException(nameof(resourcesPath));
-            _resourceName = resourceName ?? throw new ArgumentNullException(nameof(resourceName));
+            _resourceName = resourceName;
```

Comment:
I dislike this change, but I know why you did this

Reply:
It's possible to keep the check, forcing the caller to pass empty string in case it doesn't have prefix.

Reply:
I may push some commits soon ..